test/optim/test_optim.py

```diff
@@ -76,12 +76,19 @@ def _multistep_backprop_diff_hyperparams_fn(
 
     # This copy is necessary so the update on line 78 doesn't overwrite the original kwargs values
     kwargs = kwargs.copy()
+
+    # Have to pass in beta1 and beta2 seperately
+    # so they're passed in as Tensors (not a tuple) and recognized by gradcheck
+    if "beta1" in kwargs or "beta2" in kwargs:
+        assert "beta1" in kwargs and "beta2" in kwargs
```

Comment:
Wait should this assert be outside the if?

Reply:
I wanted to make sure that if someone passed in one of the beta kwargs but not the other it would raise an issue!

Reply:
Ah I missed the and vs or :p 

Could you add a clarifying message then?

Reply:
Wait lol I meant as an error, like 
```suggestion
        assert "beta1" in kwargs and "beta2" in kwargs, "Both betas should be defined in kwargs"
```